### PR TITLE
chore(example-todo): ignore coverage for geocode

### PR DESCRIPTION
**Pull request by nabdelgadir**
I'm proposing we ignore coverage for this section, because for unrelated PRs, this section will cause the coverage to decrease if the geocoder service is not up and running.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: examples/todo/src/controllers/todo.controller.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export class TodoController {
     if (todo.remindAtAddress) {
       const geo = await this.geoService.geocode(todo.remindAtAddress);
 
+      /* istanbul ignore next */
       if (!geo[0]) {
```
Comment (bajtos) [inline examples/todo/src/controllers/todo.controller.ts:51]: I am not sure if this is the right fix. Can we please add a test to verify handling of this error case instead? Adding "istanbul ignore next" would be ok if such test was too difficult to write.

Comment (nabdelgadir) [reply examples/todo/src/controllers/todo.controller.ts:51]: There is a test for this case [here](https://github.com/strongloop/loopback-next/blob/geo-coverage/examples/todo/src/__tests__/acceptance/todo.acceptance.ts#L103), but the issue is when the service is down, it won't run that test, which is causing the coverage to decrease.

Comment (bajtos) [reply examples/todo/src/controllers/todo.controller.ts:51]: I see, makes sense 👍🏻

Comment (bajtos) [reply examples/todo/src/controllers/todo.controller.ts:51]: @nabdelgadir Can you please add a code comment to capture your explanation for future readers?

Comment (nabdelgadir) [reply examples/todo/src/controllers/todo.controller.ts:53]: Added, thanks!

**Pull request merged**